### DISCUSSION TITLE
Flutter / Supabase - 'distinct' select?

### DISCUSSION THREAD

**Discussion by davoutuk** [supabase/supabase#16346] [Questions]
From my Flutter app, when querying Supabase, is there a way to apply a 'distinct(field_name)' to the select clause?

**Comment by encima** [comment]
Hi @davoutuk, while there is not a call for that within the Flutter library, you can do this using `views`. 
In a previous question, this was [answered](https://github.com/orgs/supabase/discussions/3294#discussioncomment-1374282)

The TL;DR is:
1. Create a view
`create view distinct_field  as select distinct field from table;`
2. Call it from your app
`supabase.from('distinct_field').select()`

Hope that helps!
